book_review/app/controllers/book_ratings_controller.rb

```diff
@@ -0,0 +1,44 @@
+class BookRatingsController < ApplicationController
+  before_action :set_book_rating, only: [:show, :edit, :update, :destroy]
+
+  expose :book_rating
+  expose :book, ->{ Book.find(params[:book_id]) }
```

Comment:
expose gem will do it itself, you dont need to find by param. leave only:

expose :book